eng/Versions.props

```diff
@@ -88,12 +89,12 @@
     <!-- If a System.* package is stuck on version 4.3.x, targets .NET Standard 1.x and hasn't been
     updated in years, you most likely DON'T need it, please exercise caution when adding it to the list. -->
     <SystemPackageVersionVersion>8.0.0</SystemPackageVersionVersion>
-    <SystemCollectionsImmutableVersion>$(SystemPackageVersionVersion)</SystemCollectionsImmutableVersion>
+    <SystemCollectionsImmutableVersion>10.0.0-preview.4.25223.119</SystemCollectionsImmutableVersion>
```

Comment:
Strange to see property-variable replaced with a hardcoded value, would prefer to only replace hardcoded with another hardcoded.

And not include preview versions.